wakuv2/waku.go

```diff
@@ -1090,6 +1094,24 @@ func (w *Waku) Start() error {
 		}
 	}()
 
+	w.wg.Add(1)
```

Comment:
This should be inside the condition, otherwise if `telemetryServerURL` is not set, `wg.Done()` will be never called

Reply:
good catch!